### PR TITLE
python3-SQLAlchemy: update to 1.4.50.

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)

## I checked depends from 
- https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_4_50/setup.cfg

### CODE REVIEW

File: srcpkgs/python3-SQLAlchemy/template [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 pkgname=python3-SQLAlchemy
-version=1.3.18
-revision=6
+version=1.4.50
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
-depends="python3"
+depends="python3 python3-typing_extensions python3-greenlet"
```
Comment (paper42) [inline srcpkgs/python3-SQLAlchemy/template:8]: why typing-extensions?

Comment (Luciogi) [reply srcpkgs/python3-SQLAlchemy/template:8]: https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_4_50/setup.cfg#L85

Comment (classabbyamp) [reply srcpkgs/python3-SQLAlchemy/template:8]: that's an optional dependency if using aiosqlite

Comment (Luciogi) [reply srcpkgs/python3-SQLAlchemy/template:8]: So I `typing-extensions` along with `greenlet` should be removed ?

**Pull request merged**